UPGRADE-18.0.md

```diff
@@ -551,3 +551,38 @@ module "cluster_after" {
   }
 }
 ```
+
+## Upgrade Use Case Examples
```

Comment:
this all looks great, lets just remove line 555 so it flows with the above `Before ... / After ...`

and thanks for the PR!

Reply:
No problem, and done.